e2e/cephfs_helper.go

```diff
@@ -98,6 +98,23 @@ func createCephfsSecret(f *framework.Framework, secretName, userName, userKey st
 	return err
 }
 
+func unmountCephFSVolume(f *framework.Framework, app *v1.Pod, pvc *v1.PersistentVolumeClaim) error {
+	app, err := f.ClientSet.CoreV1().Pods(f.UniqueName).Get(context.TODO(), app.Name, metav1.GetOptions{})
```

Comment:
It is not very clean to re-assign variables that were passed as an argument to a function. Using a new variable name is makes the code easier to understand.

Is the getting of the `app` Pod really needed here?

This works on our standard e2e because there is only a single node (minikube). It would be good to add a comment like `FIXME: run the unmount command on the node where the pod is running` or similar. It would give a hint on what is needed to  get addressed when we start running multi-node tests.

Reply:
> It is not very clean to re-assign variables that were passed as an argument to a function. Using a new variable name is makes the code easier to understand.

Okay, app name and volume name is sufficient. I will change the function to use them instead.

> Is the getting of the `app` Pod really needed here?

Need `UID` of `app` pod to form the mount path to run unmount command.
`cmd := fmt.Sprintf("umount /var/lib/kubelet/pods/%s/volumes/kubernetes.io~csi/%s/mount", app.ObjectMeta.UID, pv.ObjectMeta.Name)`

> This works on our standard e2e because there is only a single node (minikube). It would be good to add a comment like `FIXME: run the unmount command on the node where the pod is running` or similar. It would give a hint on what is needed to get addressed when we start running multi-node tests.

I'll do that :+1: